priv/repo/migrations/20210718173811_add_source_to_brews.exs

```diff
@@ -0,0 +1,25 @@
+defmodule BrewDash.Repo.Migrations.AddSourceToBrews do
+  use Ecto.Migration
+
+  def change do
+    alter table(:brews) do
+      add :source, :string
+      add :source_id, :string
+      add :source_recipe_id, :string
```

Comment:
I am still not convinced that we need this.

Reply:
lets chat